2020/hamming-codes/spanish/sentence_translations.json

```
@@ -713,7 +713,7 @@
  {
   "input": "If there was no error in the odd column but there was one in the right half, that tells you it's in the second to last column.",
   "model": "nmt",
-  "translatedText": "Si no hubo ningún error en la columna impar pero hubo uno en la mitad derecha, eso le indica que está en la penúltima columna.",
+  "translatedText": "Si no hubiese ningún error en la columna impar pero hubiese uno en la mitad derecha, eso indica que está en la penúltima columna.",
```

"Hubo" sounds fine to me. However, am I the only one who finds it ambiguous that the last sentence refers to "el error"? (maybe the sentence sounds confusing to me because I am a "usted" speaker and that makes "está" extra ambiguous for me?). If it is also ambiguous for other people, I would suggest:

```suggestion
  "translatedText": "Si no hubiese ningún error en la columna impar pero hubiese uno en la mitad derecha, eso indica que el error está en la penúltima columna.",
```

If just me, ignore this suggestion.